Volontulo/app/src/main/java/com/stxnext/volontulo/ui/offers/OfferEditFragment.java

```diff
@@ -0,0 +1,375 @@
+
+package com.stxnext.volontulo.ui.offers;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.support.annotation.IdRes;
+import android.support.annotation.Nullable;
+import android.support.design.widget.TextInputLayout;
+import android.support.v4.app.FragmentActivity;
+import android.text.Editable;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.ScrollView;
+import android.widget.TextView;
+import android.widget.Toast;
+import android.widget.ViewSwitcher;
+
+import com.google.android.gms.common.api.Status;
+import com.google.android.gms.location.places.AutocompleteFilter;
+import com.google.android.gms.location.places.Place;
+import com.google.android.gms.location.places.ui.PlaceSelectionListener;
+import com.google.android.gms.location.places.ui.SupportPlaceAutocompleteFragment;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.squareup.picasso.Picasso;
+import com.stxnext.volontulo.R;
+import com.stxnext.volontulo.VolontuloApp;
+import com.stxnext.volontulo.VolontuloBaseFragment;
+import com.stxnext.volontulo.api.Offer;
+import com.stxnext.volontulo.api.SaveError;
+import com.stxnext.volontulo.api.SaveResponse;
+import com.stxnext.volontulo.api.UserProfile;
+import com.stxnext.volontulo.logic.session.SessionManager;
+import com.stxnext.volontulo.ui.utils.BaseTextWatcher;
+
+import org.parceler.Parcels;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+import io.realm.Realm;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Converter;
+import retrofit2.Response;
+
+public class OfferEditFragment extends VolontuloBaseFragment {
```

Comment:
It is CP (aka copy-pasted) from `AddOfferFragment` or simply these two merged into one?